ria_remote/create_sibling_ria.py

```diff
@@ -0,0 +1,305 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Create a sibling in a configured RIA store"""
+
+__docformat__ = 'restructuredtext'
+
+
+import logging
+import subprocess
+from pathlib import Path
+from shlex import quote as sh_quote
+
+from datalad.interface.common_opts import (
+    recursion_flag,
+)
+from datalad.interface.base import (
+    Interface,
+    build_doc,
+)
+from datalad.interface.results import (
+    get_status_dict,
+)
+from datalad.interface.utils import eval_results
+from datalad.support.param import Parameter
+from datalad.support.constraints import (
+    EnsureNone,
+    EnsureStr,
+)
+from datalad.distribution.dataset import (
+    EnsureDataset,
+    datasetmethod,
+    require_dataset,
+)
+
+from ria_remote.remote import RIARemote
```

Comment:
Side note: It would be good to bury this somewhere. Ideally we can move this PR to core ASAP and deal with the special remote bits later.